frontend/src/components/Bill/Bill.jsx

```diff
@@ -44,97 +45,116 @@ const RelatedBills = ({ related }) => {
         {members.map((member) => (
           <div key={member.memberId}>
-            <span>
-              {member.fullName}:{member.memberId}
-            </span>
+            {member.fullName}: {member.memberId}
           </div>
         ))}
       </div>
-      <div>
-        <span>Total Committee Members: {members.length}</span>
-      </div>
+      <div>Total Committee Members: {members.length}</div>
     </div>
   );
 };
+
 BillCommitteeMembers.propTypes = {
   committee: PropTypes.object.isRequired,
   memberId: PropTypes.string.isRequired,
   isSenate: PropTypes.bool.isRequired,
 };
 
+// Main Bill Component
 export const Bill = () => {
   const { sessionYear, printNo } = useParams();
   const [bill, setBill] = useState();
   const [committee, setCommittee] = useState();
   const [placeName, setPlaceName] = useState('');
+  const [atBill, setATBill] = useState('');
+  const [campaign, setCampaign] = useState('');
 
   const fetched = !!bill;
+  const campaigned = !!campaign;
   const isSenate = bill?.billType?.chamber?.toLowerCase() === 'senate';
   const senateSiteUrl = `https://www.nysenate.gov/legislation/bills/${sessionYear}/${printNo}`;
 
-  // Fetch main bill data
+  // Fetch bill data
   useEffect(() => {
     const fetchBill = async () => {
       const res = await fetch(`/api/v1/bills/${sessionYear}/${printNo}`);
-      await setBill(await res.json());
+      setBill(await res.json());
     };
     fetchBill();
-  }, []);
+  }, [sessionYear, printNo]);
 
-  // Fetch bill committee data
+  // Fetch committee data
   useEffect(() => {
     if (!fetched || !isSenate) return;
+
     const chamber = bill.billType.chamber.toLowerCase();
-    const committee = bill?.status?.committeeName;
-    if (!committee) return;
+    const committeeName = bill?.status?.committeeName;
+    if (!committeeName) return;
 
     const fetchCommittee = async () => {
       const res = await fetch(
-        `/api/v1/committees/${sessionYear}/${chamber}/${committee}`
+        `/api/v1/committees/${sessionYear}/${chamber}/${committeeName}`
       );
       setCommittee(await res.json());
     };
-
     fetchCommittee();
-  }, [bill, sessionYear]);
+  }, [fetched, isSenate, bill, sessionYear]);
 
+  // Retrieve place name from session storage
   useEffect(() => {
     const setPlaceNameFromStorage = () => {
       setPlaceName(window.sessionStorage.getItem('placeName') || '[CITY, ZIP]');
     };
     setPlaceNameFromStorage();
     window.addEventListener('storage', setPlaceNameFromStorage);
+
     return () => {
       window.removeEventListener('storage', setPlaceNameFromStorage);
     };
   }, []);
 
-  if (!bill)
-    return (
-      <div>
-        <div>This is the bill {printNo}&apos;s page</div>
-      </div>
-    );
+  // Retrieve Airtable info for Bill
+  useEffect(() => {
+    const fetchATBill = async () => {
+      const res = await fetch(`/api/v1/bills/airtable-bills/${printNo}`);
+      console.log(`/api/v1/bills/airtable-bills/${printNo}`)
```

Comment:
don't forget to remove the console.logs :p

Reply:
I will eventually! I realize the printNo is not the same as the airtable Senate Bill ID (there are trailing/leading zeros in the airtable), so that's a different issue to solve before merging this one

Reply:
tis fixed!